lev.py

```diff
@@ -176,7 +176,8 @@ def update_matrix(matrix, row, col, value):
     if matrix[row][col] > value: matrix[row][col] = value
 
 # internet levensthein
-def levenshtein(str1, str2):
+# Added distance as optional parameter to avoid breaking function when imported
+def levenshtein(str1, str2, distance = 1000):

```

Comment:
@tobylunt This is the key breaking change, as previously we were assuming the script will only be invoked from the CLI. Let me know if this is alright. I've ensured that all invocations of the levenshtein() function either have options.distance or a default passed as a parameter.